module/behaviour/skills/HeadBehaviourSoccer/src/HeadBehaviourSoccer.hpp

```diff
@@ -23,18 +23,8 @@
 #include <nuclear>
 #include <set>
```

Comment:
Do we need this? Or the ones below, `message/motion/HeadCommand` and `message/vision/Ball`